store/src/java-test/com/zimbra/cs/filter/RejectTest.java

```diff
@@ -170,9 +186,9 @@ public void testNotemptyEnvelopeFromAndUsingVariables() {
             LmtpAddress recipient = new LmtpAddress("<test@zimbra.com>", null, null);
             env.setSender(sender);
             env.addLocalRecipient(recipient);
-            String raw = "From: sender@in.telligent.com\n" 
-   					+ "To: coyote@ACME.Example.COM\n"
-   					+ "Subject: test\n" + "\n" + "Hello World.";
+//            String raw = "From: sender@in.telligent.com\n" 
+//   					+ "To: coyote@ACME.Example.COM\n"
+//   					+ "Subject: test\n" + "\n" + "Hello World.";
```

Comment:
It's okay to remove this "raw" String.  The have been there unused since this RejectTest.java was initially merged.  Sorry.